### ISSUE TITLE
Remove after pausing for a while

### ISSUE THREAD

**Issue by ettom** [springjools/deluge-autoremoveplus#24]
Am I correct in understanding that currently there is no way to have finished torrents pause after seeding for a while and then get removed after some time? I tried to achieve this with both "time since completion" and "remove after seedtime" but nothing gets removed. I guess the second one is explained by https://github.com/springjools/deluge-autoremoveplus/issues/10#issuecomment-574716633, but why doesn't the first one work? Does enabling pause just completely cancel out removal options? The semantics of the interface are pretty unclear and even contradictory, it seems. Perhaps some documentation could be added?

I'm on deluge 2.0.3 with AutoRemovePlus-0.6.15

**Comment by springjools** [comment]
Yes, they are a bit confusing. Actually the plugin would need a redo. 

You can achieve a pause and removal by using the options higher up, pause after seedtime and remove after seedtime. Time since completion and those conditions only apply to unfinished torrents.
